packages/fontaine/src/index.ts

```
@@ -1,7 +1,72 @@
+
+/**
+ * Renders structured objects back into CSS strings.
+ * @type {typeof renderCSS}
+ */
+export const render = renderCSS;
+
+/**
+ * Utility function to calculate fallback values.
+ * @type {typeof calculateFallback}
+ */
+export const calculateFallback = calculateFallback;
```

ESLint

[error] 'calculateFallback' is already defined.

(ts/no-redeclare)